src/components/Reviews/LikeDislike.jsx

```diff
@@ -4,17 +4,18 @@ import { Button } from "@mui/material";
 
 const UpdownVote = ({ votes, review_id }) => {
   const [updateVote, setUpdateVote] = useState(0);
+  const [error, setError] = useState("");
 
   const upVote = (votes) => {
     setUpdateVote((currChange) => currChange + votes);
-
-    api.changeVote(review_id, votes);
+    setError("");
+    api.changeVote(review_id, votes).catch((err) => setError(err));

```

Comment:
This is great. The only thing left is to change the votes back where they were before the error. So when the uses presses like, it goes from 2 to 3. Then it shown the error message but still showing 3 likes. It should show still 2.